Servers/UI/OJS.Servers.Ui/ClientApp/src/common/types.ts

```diff
@@ -79,6 +79,13 @@ interface IRegisterForContestResponseType {
     id: number;
     name: string;
     requirePassword: boolean;
+    validationResult: IValidationType;
+}
+
+interface IException {
+    title: string;
+    status: number;
+    detail: string;
```

Comment:
Is this needed?
You can use IErrorDataType from use-http
And then casts will be unnecessary.

Reply:
I replaced IException with IErrorDataType, but the casts are unfortunately still necessary. Or is there something I am missing ?